linode/manager
# M3-5276 added Mui-disabled rules for MuiButton

## Description

Changed the colors of disabled buttons for the dark theme.

## How to test

Change to the dark theme. Make sure that when buttons are disabled they match the new spec of `#454b54` for the background and `#5c6470` for the text.
